debugger-launchjson.md

```diff
@@ -0,0 +1,58 @@
+# Configurating launch.json for C# debugging
+The launch.json file is used to configure the debugger in Visual Studio Code.
+
+Visual Studio Code generates a launch.json with almost all of the required information. To get started debugging you need to fill in the program field with the path to the executable you plan to debug. This must be specified for both the launch and attach (if you plan to attach to a running instance at any point) configurations.
+
+The generated file contains two sections. One that configures debugging for launch and a second that configures debugging for attach.
+
+If your workspace has only one launchable project, the C# extension will offer to automatically generate this file. If you missed this prompt, you can force the generation by executing the command `.NET: Generate Assets for Build and Debug` from the VS Code command pallet.
+
+# Configurating VS Code's debugging behavior
+
+## PreLaunchTask
+The `preLaunchTask` field runs the associated taskName in tasks.json before debugging your program. You can get the default build prelaunch task by executing the command `Tasks: Configure Tasks Runner` from the VS Code command pallet.
+
+This will create a task that runs `dotnet build`. You can read more about tasks at [https://code.visualstudio.com/docs/editor/tasks](https://code.visualstudio.com/docs/editor/tasks).
+
```

Comment:
We should add documentation for the other fields too. Looks like we are missing: program, args, cwd, stopAtEntry